src/app/components/sidenav/sidenav.module.ts

```diff
@@ -41,6 +43,8 @@ import { ActionDirective } from './directives/action.directive';
   imports: [
     CommonModule,
     CoreModule.forChild(),
+    CoreExtensionsModule.forRoot(),
+    ExtensionsModule.forRoot(),
```

Comment:
should not be root module
```suggestion
    ExtensionsModule.forChild(),
```